packages/react-pose/src/components/Transition/children.ts

```diff
@@ -68,11 +72,14 @@ const mergeChildren = ({
     };
 
     if (entering.has(child.key as string)) {
+      // If child is entering
       newChildProps.initialPose = preEnterPose;
-      newChildProps._pose = enterPose;
+      newChildProps._pose = enterPose; // TODO: Remove _pose and merge with child.props.pose
     } else if (moving.has(child.key as string) && flipMove) {
+      // If child is moving and we're using `flip`
       newChildProps._pose = [enterPose, 'flip'];
     } else {
+      // If child is moving
```

Comment:
previous condition checks against `moving.has(child.key as string)` to test if the child is moving, this branch of the code is unconditional, is it intended?

Reply:
It checks for that and if flipMove is true, so this does get hit if flipMove is false

Reply:
but this last branch doesnt check against `moving.has(child.key as string)` so I'm this comment looks off to me, the last `else` branch is "activated" when:
- `moving.has(child.key as string) && !flipMove`
- `!moving.has(child.key as string) && flipMove`
- `!moving.has(child.key as string) && !flipMove`

it covers 3 supplementary cases (in relation to the previous condition) and in 2 of them the child is NOT moving